control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -703,6 +703,22 @@ impl SetVolumeReplica {
     }
 }
 
+/// Set the volume property.
+#[derive(Serialize, Deserialize, Default, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct SetVolumeProperties {
+    /// The uuid of the volume.
+    pub uuid: VolumeId,
+    /// Map of property name to value.
+    pub properties: HashMap<String, String>,
```

Comment:
We can use an enum here rather than a map?

Reply:
fixed.